frontend_build/src/extract_$trs.js

```diff
@@ -0,0 +1,72 @@
+var esprima = require('esprima');
+var logging = require('./logging');
+var fs = require('fs');
+var mkdirp = require('mkdirp');
+var path = require('path');
+
+function extract$trs(messageDir, messagesName) {
+  this.messageDir = messageDir;
+  this.messagesName = messagesName;
+}
+
+extract$trs.prototype.apply = function(compiler) {
```

Comment:
It's just how Webpack plugins are done, for some reason: https://github.com/webpack/docs/wiki/how-to-write-a-plugin#basic-plugin-architecture